src/components/ids-month-view/ids-month-view.ts

```diff
@@ -1991,24 +1982,11 @@ class IdsMonthView extends Base {
             calendarEvent.cssClass = extraCss;
           }
 
-          if (!isCustom) {
-            // position event element vertically
-            calendarEvent.setAttribute(attributes.Y_OFFSET, `${(calendarEvent.order * 16) + BASE_Y_OFFSET}px`);
+          calendarEvent.setAttribute(attributes.Y_OFFSET, `${(calendarEvent.order * 16) + BASE_Y_OFFSET}px`);
```

Comment:
Can you confirm the Y_OFFSET is overriden in the Custom Event? If so, I fgured we can always set it. Knowing it will be overridden.

Reply:
Yes we are overriding the Y_OFFSET value in Custom Event, actually the Y_OFFSET value is being used to set top value for the respective event pill.